data/unpack.go

```diff
@@ -21,7 +21,7 @@ func Unpack(base string, uri string) (err error) {
 	}
 
 	if info.IsDir() {
-		os.Mkdir(base, 0777)
+		os.Mkdir(base, 0750)
```

Comment:
The installer code-base is not internally consistent on this, but personally I prefer the Go itself to try unpacking without distinguishing between owner, group, and all permissions (e.g. `0777`).  If the caller has their own preferences (e.g. they don't trust all at all, or they don't trust the group for write access), they can set their [umask][1].  But restricting them in Go means there is no way for users to set things up if, for whatever reason, they do want to give their group write access.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/umask.html

Reply:
Alright; I can disable that warning from gosec and return to the defaults.